polkapox/1.0.0-beta/Dockerfile

```diff
@@ -0,0 +1,85 @@
+##### Building stage #####
+
+# Stage 1: Build Python environment with required packages
+FROM python:3.8-slim AS app
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
+
+# Add metadata
+LABEL authors="CDC PolkaPox Team"
+LABEL description="Container used in the PolkaPox pipeline by graph_reconstruct.nf module"
+LABEL base.image="python:3.8-slim"
+LABEL dockerfile.version="1.0"
+LABEL website="https://github.com/CDCgov/polkapox"
+LABEL license="https://github.com/CDCgov/polkapox/blob/master/LICENSE"
+
+# Install required system dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential \
+    libopenblas-dev \
+    wget \
+    procps
```

Comment:
so, when I made this change and removed app2 as described above, the resulting image is twice as big (2+GB instead of 1.15GB).  I guess this is ok?  I just feel like smaller is better but I can't keep this change if the lightweight format doesn't meet the docker-builds template.

Reply:
2+GB! Woah!!! You aren't installing very much, though. I wonder why it's so large.

I have some nosey questions:

Do you know why you are installing build-essential with apt-get? 

Also, is `libgomp1` smaller than `libopenblas-dev`? I ask because the blast dockerfile has `libcomp1` (https://github.com/StaPH-B/docker-builds/blob/master/blast/2.15.0/Dockerfile). 

From what I'm aware of, you only need wget and ca-certificates for downloading blast, and procps because nextflow needs it.

Reply:
just chiming in to say that when looking at the image size reported by the `docker images` command is the uncompressed size. when the docker image is uploaded to dockerhub & quay we can see the compressed sizes which are much leaner (usually). Also it won't include the `test` stages so that will also reduce the size

would recommend removing `build-essential` if not necessary. I have a feeling it might be required for installing stuff with `pip` though...